.pre-commit-config.yaml

```diff
@@ -8,6 +8,8 @@ repos:
         exclude: ^notice/overrides.json
     -   id: check-merge-conflict
     -   id: check-yaml
+        exclude: >
```

Comment:
Why we exclude them from the linting?

Reply:
I couldn't figure out how to tell yaml-lint to allow multiple documents ina single file.  Is there an option to set to allow that?